org.eclipse.cpacep/test/org/eclipse/cpacep/util/CPACEPConnectorTest.java

```diff
@@ -0,0 +1,72 @@
+package org.eclipse.cpacep.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CPACEPConnectorTest {
+	String home = System.getProperty("user.home");
+	String tempDirectory = System.getProperty("java.io.tmpdir");
+	//String homePath = "/home/supun/Documents/cpachecker";
+	//Path homePath = Paths.get(File.separator + "home").resolve("supun").resolve("Documents").resolve("CPAchecker-1.7-unix");
+	//Path homePath = Paths.get(home, "Documents", "CPAchecker-1.7-unix");
+	Path homePath = Paths.get(home, "build", "SupunArunoda", "CPACEP","cpachecker");
```

Comment:
I've mentioned in one of the previous reviews that this will not execute locally.
But it should be possible to run unit tests not only in Travis, but also before pushing.
So first, this path should be independent of Travis. How about using a relative directory?
I think it should be ok to assume that we are in the CPACEP directory, so `./cpachecker` should do.
Second, this unit test expects that CPAchecker was cloned into `./cpachecker` before.
Users will not know this, and when executing all unit tests from within an IDE, this will always fail.

For now, to quickly resolve this PR and get a working travis, I would accept only having the relative directory
and to ignore the second issue, but we should create a github issue for it.